### PR TITLE
RavenDB-13395 Copy Button for Change Vector

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts [hunk 1]
```diff
@@ -582,6 +576,10 @@ class editDocument extends viewModelBase {
         copyToClipboard.copy(this.editedDocId(), "Document ID has been copied to clipboard");
     }
 
+    copyChangeVectorToClipboard() {
+        copyToClipboard.copy(_.replace(this.changeVectorFormatted(), new RegExp("<br/> ", "g"), "\n"), "Change Vector has been copied to clipboard");
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts]: can we avoid tying this to html form, instead we can do:
this.changeVector().map(vectorItem => vectorItem.fullFormat).join(' ');

File: src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html [hunk 2]
```diff
@@ -159,34 +159,36 @@
         </div>
     </div>
     <div id="right-options-panel" class="flex-window">
-        <div class="panel document-properties flex-window-head">
+        <div class="panel document-properties flex-window-head" id="document-properties-area">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html]: we already have nice selector for that: document-properties no need for new one.

File: src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html [hunk 3]
```diff
@@ -159,34 +159,36 @@
...
                     <div data-bind="visible: !$root.isCreatingNewDocument()">
                         <dt>Change Vector:</dt>
                         <dd>
-                            <span data-placement="bottom" data-toggle="tooltip" data-html="true" data-animation="true"
-                                  data-bind="attr: { 'data-original-title': $root.changeVectorHtml() }">
-                                    <span data-bind="foreach: $root.changeVector">
-                                        <span class="badge badge-default" data-bind="text: shortFormat"></span>
-                                    </span>
-                             </span>
+                            <span data-placement="left" data-toggle="tooltip" data-html="true" data-animation="true" data-bind="tooltipText: $root.changeVectorHtml()" id="changeVector">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html]: do we need id="changeVector" here?

File: src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts]: done

File: src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html]: removed

File: src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html]: done

**Review by ml054 — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**